### PR TITLE
add rb-pastel

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@sodre @adbeda agree to be maintainers?

**Comment by rluria14** [comment]
@conda-forge/help-ruby

File: recipes/rb-pastel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: f577708463313119d2f7303eefb24217879d188e94f2eea8a19373fdcee48354
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install --norc -l -V --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
```
Comment (sodre) [inline recipes/rb-pastel/meta.yaml]:
```suggestion
      && gem unpack {{ name }}-{{ version }}.gem
```

File: recipes/rb-pastel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+    - rb-equatable >=0.6.0,<0.7.0
```
Comment (sodre) [inline recipes/rb-pastel/meta.yaml]: the gem requirement was `~> 0.6`, which maps to `>=0.6,<1`
```suggestion
    - rb-equatable >=0.6,<1
```

File: recipes/rb-pastel/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - rb-tty-color >=0.5.0,<0.6.0
```
Comment (sodre) [inline recipes/rb-pastel/meta.yaml]: Same here, the original requirement was `~>0.5`
```suggestion
    - rb-tty-color >=0.5,<1
```

**Comment by rluria14** [comment]
@sodre changes have been committed

**Comment by rluria14** [comment]
> @sodre changes have been committed

@conda-forge/help-ruby
@conda-forge/staged-recipes

ready for final review and merge.

**Pull request merged**